scripts/badger/aura_tcl_strategy.py

```diff
@@ -0,0 +1,136 @@
+from decimal import Decimal
+from brownie import ZERO_ADDRESS, interface
+from rich.console import Console
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+console = Console()
+
+# flag
+prod = False
+
+# slippages
+SLIPPAGE = 0.995
+COEF = 0.9825
+
+# breakdowns of rewards
+AURA_FOR_STABLES = 0.3
+BAL_FOR_STABLES = 0.7
+
+
+def main():
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    voter = GreatApeSafe(r.badger_wallets.treasury_voter_multisig)
+    vault.init_aura()
+    vault.init_balancer()
+    vault.init_cow(prod)
+
+    # tokens involved
+    usdc = vault.contract(r.treasury_tokens.USDC)
+    bal = vault.contract(r.treasury_tokens.BAL)
+    weth = vault.contract(r.treasury_tokens.WETH)
+    aura = vault.contract(r.treasury_tokens.AURA)
+    b80bal_20weth = vault.contract(r.balancer.B_80_BAL_20_WETH)
+    bauraBAL_stable = vault.contract(r.balancer.B_auraBAL_STABLE)
+    auraBAL = vault.contract(r.treasury_tokens.AURABAL)
+    bauraBal = vault.contract(r.sett_vaults.bauraBal)
+    graviaura = vault.contract(r.sett_vaults.graviAURA)
+    bvecvx = vault.contract(r.sett_vaults.bveCVX)
+    bcvxcrv = vault.contract(r.sett_vaults.bcvxCRV)
+
+    # contracts
+    wrapper = vault.contract(r.aura.wrapper)
+    registry_v_2 = vault.contract(r.registry_v2, interface.IBadgerRegistryV2)
+
+    # snaps
+    vault.take_snapshot([usdc, auraBAL, bauraBal, graviaura])
+    voter.take_snapshot([aura])
+
+    # 1. claim rewards
+    vault.aura.claim_all_from_booster()
+
+    # 2. organised splits for each asset
+    balance_bal = bal.balanceOf(vault)
+    balance_aura = aura.balanceOf(vault)
+    console.print(
+        f"[green] === Claimed rewards {balance_bal/1e18} BAL and {balance_aura/1e18} AURA === [/green]"
+    )
+
+    bal_swap_for_usdc = balance_bal * BAL_FOR_STABLES
+    aura_swap_for_usdc = balance_aura * AURA_FOR_STABLES
```

Comment:
```suggestion
    bal_swap_for_usdc = int(balance_bal * BAL_FOR_STABLES)
    aura_swap_for_usdc = int(balance_aura * AURA_FOR_STABLES)
```
this prevents rounding problems which might arise in L70 and L72